rotkehlchen/tests/fixtures/rotkehlchen.py

```diff
@@ -133,10 +137,23 @@ def patch_and_enter_before_unlock(
 ) -> None:
     # Do not connect to the usual nodes at start by default. Do not want to spam
     # them during our tests. It's configurable per test, with the default being nothing
-    rpc_nodes_result = ethereum_manager_connect_at_start + optimism_manager_connect_at_start if network_mocking is False else []  # noqa: E501
+    def mock_get_rpc_nodes(blockchain, only_active):  # pylint: disable=unused-argument
```

Comment:
So .. before what was happening was that `db.get_rpc_nodes()` was returning "optimism + ethereum" nodes to connect to for **any** chain. yes ... lol ...